lib/generators/boring/devise/jwt/install/install_generator.rb

```diff
@@ -0,0 +1,224 @@
+          return if File.exist?("app/models/#{options[:model_name].underscore}.rb")
+
+          say "We couldn't find the #{options[:model_name]} model. Maybe there is a typo? Please provide the correct model name and run the generator again.", :red
+
+          abort
+        end
+
+        def add_devise_jwt_gem
+          say "Adding devise-jwt gem", :green
+          gem "devise-jwt"
+        end
+
+        def add_devise_jwt_config_to_devise_initializer
+          say "Adding devise-jwt configurations to a file `config/initializers/devise.rb`", :green
+
+          jwt_config = <<~RUBY
+            config.jwt do |jwt|
+              jwt.secret = #{devise_jwt_secret_key}
+              jwt.dispatch_requests = [
+                ['POST', %r{^/sign_in$}]
+              ]
+              jwt.revocation_requests = [
+                ['DELETE', %r{^/sign_out$}]
+              ]
+              jwt.expiration_time = #{options[:expiration_time_in_days]}.day.to_i
+            end
+          RUBY
+
+          inject_into_file "config/initializers/devise.rb",
+                            optimize_indentation(jwt_config, 2),
+                            before: /^end\s*\Z/m
+
+          say "❗️❗️\nValue for jwt.secret will be used from `#{devise_jwt_secret_key}`. You can change this values if they don't match with your app.\n",
+              :yellow
+        end
+
+        def configure_revocation_strategies
+          say "Configuring #{options[:revocation_strategy]} revocation strategy",
+              :green
+
+          case options[:revocation_strategy]
+          when "JTIMatcher"
+            configure_jti_matcher_strategy
+          when "Denylist"
+            configure_denylist_strategy
+          when "Allowlist"
+            configure_allowlist_strategy
+          end
+        end
+
+        private
+
+        def devise_jwt_secret_key
+          if options[:use_env_variable]
+            "ENV['DEVISE_JWT_SECRET_KEY']"
+          else
+            "Rails.application.credentials.devise_jwt_secret_key"
+          end
+        end
+
+        def configure_jti_matcher_strategy
+          model_name = options[:model_name].underscore
+          Bundler.with_unbundled_env do
+            run "bundle exec rails generate migration add_jti_to_#{model_name.pluralize}"
+          end
+          migration_content = <<~RUBY
+            add_column :users, :jti, :string, null: false
+            add_index :users, :jti, unique: true
+          RUBY
+
+          inject_into_file Dir["db/migrate/*_add_jti_to_#{model_name.pluralize}.rb"][0],
+                           optimize_indentation(migration_content, 4),
+                           after: /def change\n/,
+                           verbose: false
+
+          add_devise_jwt_module(
+            strategy: "self",
+            include_content: "include Devise::JWT::RevocationStrategies::JTIMatcher"
+          )
+                            
+        end
+
+        def configure_denylist_strategy
```

Comment:
Ditto. I think easier to just add templates for all migrations you want for denylist.